src/prommis/uky/costing/location_factors.json

```
@@ -0,0 +1,1694 @@
+            "max": 0.95,
+            "average": 0.935
+        }
+    },
+    {
+        "country": "Lebanon",
+        "city": null,
+        "location_factor": {
+            "min": 0.9,
+            "max": 0.95,
+            "average": 0.925
+        }
+    },
+    {
+        "country": "Libya",
+        "city": null,
+        "location_factor": {
+            "min": 0.91,
+            "max": 0.94,
+            "average": 0.925
+        }
+    },
+    {
+        "country": "Lithuania",
+        "city": null,
+        "location_factor": {
+            "min": 0.91,
+            "max": 0.96,
+            "average": 0.935
+        }
+    },
+    {
+        "country": "Madagascar",
+        "city": null,
+        "location_factor": {
+            "min": 0.88,
+            "max": 0.94,
+            "average": 0.91
+        }
+    },
+    {
+        "country": "Malawi",
+        "city": null,
+        "location_factor": {
+            "min": 0.89,
+            "max": 0.93,
+            "average": 0.91
+        }
+    },
+    {
+        "country": "Malaysia",
+        "city": null,
+        "location_factor": {
+            "min": 0.83,
+            "max": 0.92,
+            "average": 0.875
+        }
+    },
+    {
+        "country": "Mali",
+        "city": null,
+        "location_factor": {
+            "min": 0.87,
+            "max": 0.94,
+            "average": 0.905
+        }
+    },
+    {
+        "country": "Malta",
+        "city": null,
+        "location_factor": {
+            "min": 0.9,
+            "max": 0.95,
+            "average": 0.925
+        }
+    },
+    {
+        "country": "Mexico",
+        "city": null,
+        "location_factor": {
+            "min": 0.84,
+            "max": 0.92,
+            "average": 0.88
+        }
+    },
+    {
+        "country": "Mongolia",
+        "city": null,
+        "location_factor": {
+            "min": 0.89,
+            "max": 0.93,
+            "average": 0.91
+        }
+    },
+    {
+        "country": "Morocco",
+        "city": null,
+        "location_factor": {
+            "min": 0.89,
+            "max": 0.95,
+            "average": 0.92
+        }
+    },
+    {
+        "country": "Mozambique",
+        "city": null,
+        "location_factor": {
+            "min": 0.88,
+            "max": 0.94,
+            "average": 0.91
+        }
+    },
+    {
+        "country": "Myanmar",
+        "city": null,
+        "location_factor": {
+            "min": 0.85,
+            "max": 0.93,
+            "average": 0.89
+        }
+    },
+    {
+        "country": "Namibia",
+        "city": null,
+        "location_factor": {
+            "min": 0.88,
+            "max": 0.94,
+            "average": 0.91
+        }
+    },
+    {
+        "country": "Nepal",
+        "city": null,
+        "location_factor": {
+            "min": 0.89,
+            "max": 0.94,
+            "average": 0.915
+        }
+    },
+    {
+        "country": "Netherlands",
```

To be consistent with the table.

```suggestion
        "country": "The Netherlands",
```